frontend/src/app/testQueue/TestCard/TestCard.test.tsx

```diff
@@ -114,214 +107,167 @@ async function getSpecimenTypeDropdown() {
   )) as HTMLSelectElement;
 }
 
-describe("TestCard", () => {
-  let nowFn = Date.now;
-  let store: MockStoreEnhanced<unknown, {}>;
-  let alertSpy: jest.SpyInstance;
-  const mockStore = configureStore([]);
-  const trackEventMock = jest.fn();
-  const removePatientFromQueueMock = jest.fn();
-  const trackMetricMock = jest.fn();
-  const trackExceptionMock = jest.fn();
-
-  const testOrderInfo: QueriedTestOrder = {
-    internalId: "1b02363b-ce71-4f30-a2d6-d82b56a91b39",
-    dateAdded: "2022-11-08 13:33:07.503",
-    symptoms:
-      '{"64531003":"false","103001002":"false","84229001":"false","68235000":"false","426000000":"false","49727002":"false","68962001":"false","422587007":"false","267036007":"false","62315008":"false","43724002":"false","36955009":"false","44169009":"false","422400008":"false","230145002":"false","25064002":"false","162397003":"false"}',
-    symptomOnset: null,
-    noSymptoms: true,
-    deviceType: {
+const facilityInfo: QueriedFacility = {
```

Comment:
is this `facilityInfo` a duplicate of the [`facilityInfo`](https://github.com/CDCgov/prime-simplereport/pull/7792/files#diff-eb68b11555006268ed4f5c502b29cf4d76f8453e34e522b9fd99884c1f249f8bR296) defined in `testCardTestConstants.ts`?

Reply:
yep, good catch! for some reason earlier I was having cross-file import issues for this variable where importing it from the constants file it used to live in was causing errors, but it seems like it's working fine now 🤷